test/Microsoft.AspNetCore.Mvc.TestCommon/project.json

```diff
@@ -1,11 +1,27 @@
 {
-  "shared": "**/*.cs",
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk"
+  },
   "frameworks": {
     "net451": {},
-    "netstandard1.3": {
+    "netstandard1.6": {
       "dependencies": {
-        "NETStandard.Library": "1.6.1-*"
+        "System.ComponentModel.Annotations": "4.3.0"
```

Comment:
This was just necessary. Think we use some DataAnnotationAttributes that required this.